starlark/int_test.go

```diff
@@ -140,16 +140,16 @@ func intfallback() {
 }
 
 // The --entry flag invokes an alternate entry point, for use in subprocess tests.
+var testEntry = flag.String("entry", "", "child process entry-point")
+
 func TestMain(m *testing.M) {
-	var entry string
-	flag.StringVar(&entry, "entry", "", "child process entry-point")
 	flag.Parse()
```

Comment:
Let's add a comment:

// In some build systems, notably Blaze), flag.Parse is called before TestMain,
// in violation of the TestMain contract, making this second call a no-op.

Reply:
Added.